deployment/terraform/loadbalancing.tf

```diff
@@ -219,7 +222,7 @@ resource "aws_alb_target_group" "marlowe_dash" {
   vpc_id   = aws_vpc.plutus.id
 
   health_check {
-    path = "/version"
+    path = "/"
```

Comment:
in some cases the heathcheck is on `/` in others it's a `/version` endpoint? :thinking:

Reply:
some services have a `/health` endpoint, some have a `/version` and others don't. we just pick the most suitable endpoint for each service.